### ISSUE TITLE
require STATE?

### ISSUE THREAD

**Issue by zmanion** [CVEProject/automation-working-group#41]
Should STATE be required? The current v4 min schema does not enforce it, and I there is some ongoing discussion about states, but we could at least enforce the current states.

When submitting records, we're changing state from RESERVED to PUBLIC, is that appropriate?

**Comment by kernelsmith** [comment]
we are also changing RESERVED to PUBLIC, but admittedly I wasn't entirely sure I should be doing that either

**Comment by theall38103** [comment]
Yes, I would recommend changing it when submitting new records. Including STATE in the JSON with a value other than PUBLIC will likely introduce confusion and delays when we process a submission.

**Comment by ncaron-mitre** [comment]
I know this issue has laid fallow for a bit, but I would like to bring it back up. In terms of JSON validation, it becomes much more difficult to determine which schema should be used to validate a particular json file without the guarantee that this field should exist. Making this mandatory would simplify this issue.

**Comment by mprpic** [comment]
This issue is being closed as obsolete. If you feel it is still relevant to any currently running AWG initiatives, feel free to re-open it or open a new issue. Thank you!

**Comment by zmanion** [comment]
![cve_record_states dot](https://github.com/CVEProject/automation-working-group/assets/18131334/0d84c6e0-44f1-45d4-89c6-f9d3ac1182f4)
![cve_id_states dot](https://github.com/CVEProject/automation-working-group/assets/18131334/cb33aa4d-cc3f-4c34-9f9a-5e4047ef7152)

FWIW, state is essentially managed by CVE Services and I don't think there is much risk or ability for a CNA to improperly change the state field.
